### PR TITLE
New frontend for faucet

### PR DESCRIPTION
**Pull request by vwakesahu**

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -10,89 +9,105 @@ function App() {
...
-            <div className="bg-slate-800 h-screen">
-                <div className="flex min-h-full flex-col justify-center py-12 sm:px-6 lg:px-8">
-                    <div className="sm:mx-auto sm:w-full sm:max-w-md">
-                        <AlertView message={error} type="error"/>
-                    </div>
-                </div>
-            </div>
-        )
-    }
+  //   if (error != null) {
```
Comment (silasdavis) [inline frontend/src/App.js:56]: Where did our error handling go?

File: frontend/public/index.html [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="https://cdn.prod.website-files.com/671156d33ac264346e223043/671af30cbd385810328a2bea_favi%20(19).png" />
```
Comment (silasdavis) [inline frontend/public/index.html]: Please use the locally hosted version not a separate CDN, we will get caching via cloudflare

**Review by silasdavis — CHANGES_REQUESTED** [review]
It looks pretty!

I'd like to see some error handling back... ideally some kind of persistent toast or something.

File: frontend/src/App.js (hunk 1 above)
Comment (vwakesahu) [reply frontend/src/App.js:56]: image
I moved the error handling from App.js to IncoFaucet.js.

File: frontend/public/index.html (hunk 2 above)
Comment (vwakesahu) [reply frontend/public/index.html]: Actually, there was a problem with local images; they were not rendering when I tried build them using Docker Compose. That's why I was using a separate CDN.

**Comment by vwakesahu** [comment]
@silasdavis I removed the CDNs and am now using local images. Let me know if any further changes are required.

File: frontend/src/App.js (hunk 1 above)
Comment (silasdavis) [reply frontend/src/App.js:56]: Okay let's just delete this code rather than leaving it commented

File: frontend/src/App.js [hunk 3]
```diff
@@ -10,89 +9,106 @@ function App() {
...
-                    handleSelectChain={setSelected}
-                    address={address}
-                    onAddressChanged={setAddress}
-                    handleSubmit={handleSubmit}
-                />
-            </div>
-        </div>
-    );
+  return (
+    // <div className="bg-slate-800 h-screen">
```
Comment (silasdavis) [inline frontend/src/App.js:69]: Also delete this section

**Review by silasdavis — CHANGES_REQUESTED** [review]
Looks good to me, if you could just remove the unused code we are good to merge.

**Comment by silasdavis** [comment]
Second thoughts I'll handle this since I need to bump version anyway, nice work @vwakesahu

**Pull request merged**